app.json

```diff
@@ -35,7 +35,9 @@
     "DW_USER": {
       "required": true
     },
-    "OPEN_SOURCE": "https://api.data.world/v0/sql/citizenlabs/grb-storm-drains-test"
```

Comment:
`grb-storm-drains-test` is an intentionally smaller data set for review apps.

Reply:
The test set is to small, only has part of GR.  Need to cover multiple communities to properly evaluate data loading process.